src/main/java/net/teamterminus/machineessentials/energy/electric/template/ElectricDeviceBlockEntity.java

```diff
@@ -21,49 +21,49 @@ public void tick() {
         super.tick();
         //reset counters
         ampsUsing = 0;
-        averageAmpLoad.increment(world,0);
-        averageEnergyTransfer.increment(world,0);
+        averageAmpLoad.increment(world, 0);
+        averageEnergyTransfer.increment(world, 0);
 
         //try to pull max allowed current from any connected side
         for (Direction dir : Direction.values()) {
             BlockEntity be = MachineEssentials.getBlockEntity(dir, world, this);
-            if(be instanceof ElectricWire){
-                receiveEnergy(dir,getMaxInputAmperage());
+            if (be instanceof ElectricWire) {
+                receiveEnergy(dir, getMaxInputAmperage());
             }
         }
     }
 
     @Override
     public long receiveEnergy(@NotNull Direction dir, long amperage) {
```

Comment:
Oh my god why there's so many IFs? This is such big nest of IFs

Reply:
In this method I mean. Just look at the end of the method there's at least like 10 closing brackets before `return` and the end of the method

Reply:
> @MartinSVK12 's problem.

I've fixed it and could've push but right now I am not at my pc so ehhh